src/cljs/athens/views.cljs

```diff
@@ -86,7 +88,13 @@
 
          :else [:<>
                 (when @modal [filesystem/window])
-                [:div (stylefy/use-style app-wrapper-style)
+                [:div (use-style app-wrapper-style
+                                 {:class (join " "
+                                               [(cond
+                                                  (= (get-os) :windows) "os-windows"
+                                                  (= (get-os) :mac) "os-mac"
+                                                  (= (get-os) :linux) "os-linux")
+                                                (when (electron?) "is-electron")])})
```

Comment:
Need a better solution to derefing events that are defined only when `electron.cljs` is required via `:main` build, and not browser `:app` build.